TYPO3.Neos/Resources/Public/JavaScript/Content/Application.js

```diff
@@ -406,6 +406,12 @@ function(
 						$('title').html($htmlDom.filter('title').html());
 						$('link[rel="neos-site"]').attr('href', $htmlDom.filter('link[rel="neos-site"]').attr('href'));
 
+						var $pageStylesheets = $('link[rel="stylesheet"]:not(.neos-ui-asset)');
+						$pageStylesheets.remove();
+
+						var $loadedStylesheets = $htmlDom.filter('link[rel="stylesheet"]:not(.neos-ui-asset)');
+						$('head').append($loadedStylesheets);
```

Comment:
it's by no means any joke, there could easily be undesired side-effects of this change that would affect every page change whereas that problem is a rare one that happens occasionally